server/service/src/sync/translations/stock_line.rs

```diff
@@ -183,4 +203,43 @@ mod tests {
             assert_eq!(translation_result, record.translated_record);
         }
     }
+
+    #[actix_rt::test]
+    async fn test_stock_line_push_merged() {
+        // The item_links_merged function will merge ALL items into item_a, so all stock_lines should have an item_id of "item_a" regardless of their original item_id.
+        let (_, connection, _, _) = setup_all(
+            "test_stock_line_push_item_link_merged",
+            MockDataInserts::none()
+                .units()
+                .items()
+                .item_links_merged()
+                .names()
+                .stores()
+                .locations()
+                .barcodes()
+                .stock_lines(),
+        )
+        .await;
+
+        let repo = ChangelogRepository::new(&connection);
+        let changelogs = repo
+            .changelogs(
+                0,
+                1_000_000,
+                Some(ChangelogFilter::new().table_name(ChangelogTableName::StockLine.equal_to())),
+            )
+            .unwrap();
+
+        let translator = StockLineTranslation {};
+        for changelog in changelogs {
+            // Translate and sort
+            let Some(translated) = translator
+                .try_translate_push_upsert(&connection, &changelog)
+                .unwrap() else {
+                    panic!("Where is my stuff");
```

Comment:
I think unwrap() is enough, can do two unwrap() (one for Option)

Reply:
6a8a6a3b7833031bbae4f701a26ee353af1bdb49